frontends/web/src/api/account.ts

```diff
@@ -233,7 +252,15 @@ export interface IBalance {
     incoming: IAmount;
 }
 
-export const getBalance = (code: AccountCode): Promise<IBalance> => {
+export type BalanceResponse = {
+  success: true;
+  balance: IBalance;
+} | {
+  success: false;
+  errorMessage: string
```

Comment:
it seems the frontend does not even need this? Maybe for simplicity just remove it and the corresponding backend code that sets error codes/messages for this call?

Could consider doing the same for the other calls where it seems the frontend does not actually need the code/msg.

Reply:
Done, thanks